wdl-ast/src/v1/workflow.rs

```diff
@@ -529,7 +529,7 @@ impl AstNode for CallAfter {
 
 /// Represents an input item in a call statement.
 #[derive(Clone, Debug, PartialEq, Eq)]
-pub struct CallInputItem(SyntaxNode);
+pub struct CallInputItem(pub(crate) SyntaxNode);
```

Comment:
I don't think this is needed any longer.

```suggestion
pub struct CallInputItem(SyntaxNode);
```

Reply:
Thanks! Missed that one when reverting changes.